components/parking/ParkingBasicInfo.tsx

```diff
@@ -22,27 +74,19 @@ const ParkingBasicInfo: React.FC<ParkingBasicInfoProps> = ({ parkingLot }) => {
       </SectionAppForm>
       <SectionAppForm title={"Traffic"} icon={"car"}>
         <View style={styles.textWrapper}>
-          <DetailText text={"1st floor"} textStyle={styles.text} />
-          <DetailText text={"20/20"} />
-        </View>
-        <View style={styles.textWrapper}>
-          <DetailText text={"2st floor"} textStyle={styles.text} />
-          <DetailText text={"19/20"} />
-        </View>
-        <View style={styles.textWrapper}>
-          <DetailText text={"3st floor"} textStyle={styles.text} />
-          <DetailText text={"20/20"} />
-        </View>
-        <View style={styles.textWrapper}>
-          <DetailText text={"4st floor"} textStyle={styles.text} />
-          <DetailText text={"11/12"} />
```

Comment:
We should still show the availability of each floor no?

Reply:
<DetailText text={"availabilty"} textStyle={styles.text} />
          <DetailText
            text={`${slots.length - available_slots_count} / ${slots.length}`}
          />